src/main/kotlin/com/sourcegraph/cody/agent/CodyAgentServer.kt

```diff
@@ -72,4 +72,18 @@ interface CodyAgentServer {
   fun completionAccepted(logID: CompletionItemParams)
 
   @JsonNotification("$/cancelRequest") fun cancelRequest(cancelParams: CancelParams)
+
+  // Webviews
+  @JsonRequest("webview/didDispose") fun webviewDidDispose(): CompletableFuture<Void?>
+
+  @JsonNotification("webview/receiveMessage")
+  fun webviewReceiveMessage(params: WebviewReceiveMessageParams)
+
+  @JsonRequest("command/execute")
+  fun commandExecute(params: CommandExecuteParams): CompletableFuture<Any?>
```

Comment:
Some of these methods are not used (IMO we should avoid introducing them in advance to main branch but you can ignore my comment).

Reply:
I'd rather keep them. They may look redundant for now but the file remains clean and readable. I assume that we will need them soon for commands (migrated from recipes) and for multiple chat handling (`webview/didDispose`).